packages/sku/src/entry/client/index.tsx

```diff
@@ -8,6 +8,7 @@ import clientContextKey from '../clientContextKey.js';
 
 if (process.env.NODE_ENV === 'development') {
   if (typeof client !== 'function') {
+    // eslint-disable-next-line
```

Comment:
Can this have an explanation as well please?

Reply:
Yeah I'll add it in. 

ESM doesn't want require however this file is being loaded in via webpack which needs the require. I can probably refactor this to import `dedent` at the top and use it that way.